src/Microsoft.DocAsCode.Dfm/DfmFallbackHelper.cs

```diff
@@ -34,20 +34,24 @@ public static class DfmFallbackHelper
 
             // var currentFileFolder = Path.Combine(context.GetBaseFolder(), Path.Combine(context.GetFilePathStack().Select(path => Path.GetDirectoryName(path)).ToArray()));
             // var originalFilePath = Path.Combine(context.GetBaseFolder(),  orginalRelativePath);
-            var filePathToDocset = relativePath;
+            RelativePath filePathToDocset = null;
             string parentFileDirectoryToDocset = context.GetBaseFolder();
             var parents = context.GetFilePathStack();
             if(parents != null)
             {
                 var parent = parents.Peek();
-                filePathToDocset = ((RelativePath)parent + (RelativePath)filePathToDocset).RemoveWorkingFolder();
+                filePathToDocset = ((RelativePath)parent + (RelativePath)relativePath).RemoveWorkingFolder();
                 parentFileDirectoryToDocset = Path.GetDirectoryName(Path.Combine(context.GetBaseFolder(), parent));
             }
 
-            var originalFilePath = Path.Combine(context.GetBaseFolder(), filePathToDocset);
-            var actualFilePath = originalFilePath;
+            var originalFullPath = Path.Combine(context.GetBaseFolder(), filePathToDocset);
+            string actualFilePath = null;
             bool hitFallback = false;
-            if (!EnvironmentContext.FileAbstractLayer.Exists(originalFilePath))
+            if (EnvironmentContext.FileAbstractLayer.Exists(filePathToDocset))
```

Comment:
path to docset?  can we change it to path to repo root? otherwise i we use it in plugins? in plugins we should not have docset logic

Reply:
docfx do not know path to repo.